### DISCUSSION TITLE
flashgeotext dependecy

### DISCUSSION THREAD

**Discussion by LuciaTajuelo** [explosion/spaCy#7715] [Help: Installation] [answered]
Hi!

I want to use spacy in a project where i use flashgeotext 0.3.0 which give me a dependency problem when installing en_core_web_sm.

How should i aprroach this issue? are you planning to upgrade the model flashgeotext  dependency?

Log trace:
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
flashgeotext 0.2.0 requires pydantic==1.4, but you have pydantic 1.7.3 which is incompatible.

- Operating System:
- Python Version Used: 3.7
- spaCy Version Used: 3.0
Thanks!

**Comment by polm** [comment, marked as answer]
I don't completely understand what you're asking, but it looks like this is a conflict in dependency requirements between flashgeotext (which we do not maintain) and spaCy. For reference here's the flashgeotext repo:

https://github.com/iwpnd/flashgeotext

It looks like flashgeotext requires an older version pydantic than you have installed. Pinning to a single version is kind of unusual, maybe you can ping the flashgeotext devs about upgrading? 

I'm not familiar enough with pydantic to say for sure, but it seems like it might also be possible that despite the version requirements flashgeotext would just work with the newer version, so trying that might be worth a shot.
